examples/rpt/count_calculation/rpt_count_calculation.prm

```diff
@@ -0,0 +1,37 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# RPT Monte Carlo technique
+#---------------------------------------------------
+subsection rpt parameters
+    set particle positions file          = positions1.particle
+    set export counts                    = true
+    set iteration number                 = 1000000
+    set random number seed               = auto
+    set reactor radius       			 = 0.1
+    set peak-to-total ratio  			 = 0.4
+    set sampling time        			 = 1
+end
+	
+# --------------------------------------------------
+# Initial value of parameters for tuning
+#---------------------------------------------------
+subsection parameter tuning
```

Comment:
I would call this section parameters or rpt parameters or particle and detector parameters because these parameters will be  use for the dictionnary generation also. The tuning itself (if it becomes inside lethe) could go in another section :).

Reply:
Actually I did it because I thought we would tune all of those 5 parameters. If I keep tau, R, and mu_r in this section and move the other two to the section above (maybe mu_d to the detector section), is it better?

Reply:
I think moving the gama-rays emitted to rpt parameters, and mu_d to detector parameters is a good idea!

Reply:
I think it's better to keep all "tunable parameters" in the same section. Maybe just rename the section to use a better name?